### PR TITLE
Productivity Suite - Style/quality of life improvements

### PR DESCRIPTION
**Pull request by dario-piotrowicz**
Some small quality of life improvements:
- add missing deploy script for news fragment
- remove app-footer's fixed positioning
- improve news css and styling (to make it look more like the hackernews one and to be font-responsive)
- generic styles improvements

### CODE REVIEW

File: productivity-suite/app/fragments/news/src/root.css [hunk 1]
```diff
@@ -1,129 +1,37 @@
 piercing-fragment-host[fragment-id="news"] {
-  font-size: 15px;
-  background-color: #f2f3f5;
+  font-size: 0.935rem;
   color: #34495e;
-  overflow-y: scroll;
+  font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen,Ubuntu,Cantarell,"Fira Sans","Droid Sans","Helvetica Neue",sans-serif;
+  -webkit-font-smoothing: auto;
+  font-weight: 400;
```
Comment (dario-piotrowicz) [inline productivity-suite/app/fragments/news/src/root.css:6]: this makes the news fragment look more like the original hackernews app

File: productivity-suite/app/legacy-app/index.html [hunk 2]
```diff
@@ -159,20 +159,32 @@
         border-radius: 1px;
       }
 
+      piercing-fragment-host {
+        max-width: 80vw;
```
Comment (dario-piotrowicz) [inline productivity-suite/app/legacy-app/index.html:163]: thanks to this we don't get the fragment outlines overlap the legacy app ones on smaller screens

Comment (dario-piotrowicz) [reply productivity-suite/app/legacy-app/index.html:163]: and in any case avoid effects like this:
Screenshot 2022-11-04 at 17 42 26

**Review by petebacondarwin — APPROVED** [review]
I haven't tried running this, but the code changes look OK

**Comment by dario-piotrowicz** [comment]
> I haven't tried running this, but the code changes look OK

Awesome thanks! 😃

**Review by jculvey — APPROVED** [review]

**Pull request merged**